Gemfile.lock

```diff
@@ -230,7 +230,7 @@ GEM
     listen (3.0.8)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
-    loofah (2.5.0)
+    loofah (2.7.0)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.5.0 → 2.7.0

[_[change-log](https://github.com/flavorjones/loofah/blob/master/CHANGELOG.md), [source-code](https://github.com/flavorjones/loofah)_]





<details>
<summary>Commits</summary>

A change of **20** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.5.0...v2.7.0).

These are the first 10 commits:
- (333e666) [add border keywords](https://github.com/flavorjones/loofah/commit/333e666e61bfb6edf42f2e858b1a4f426f768e91)
- (296da9b) [packaging: update gemspec changelog URL](https://github.com/flavorjones/loofah/commit/296da9b47bfdf4cd2e426e314f0fa8686490120f)
- (deb3fa3) [maintenance: use hoe-markdown to manage markdown files](https://github.com/flavorjones/loofah/commit/deb3fa3e7c22831a0b051280ec2cfee49c13aa76)
- (6c329e8) [ci: update github icons](https://github.com/flavorjones/loofah/commit/6c329e861afb8a47e5098a5bb0a1a8d638ae529a)
- (b519a4a) [order alphabeticaly](https://github.com/flavorjones/loofah/commit/b519a4a85fab0b61d0bc3db28a6ef8ceaf1dfcf7)
- (e14b903) [Merge pull request #188 from pipefy/add-css-keywords](https://github.com/flavorjones/loofah/commit/e14b903cafe5bd2beccf0b6fca80c94d338b632b)
- (9ac6450) [ci: remove serial group on PR pipeline](https://github.com/flavorjones/loofah/commit/9ac6450bb09db54911ea9727de42eb90e3c5c49f)
- (93289f2) [dep: update to hoe-markdown ~> v1.2](https://github.com/flavorjones/loofah/commit/93289f2876bd097076bb7703fd57603f565767fe)
- (c3d8a6d) [update CHANGELOG](https://github.com/flavorjones/loofah/commit/c3d8a6d1dcfa96dc94f61221e0fa1f2f6b9e522b)
- (4e8a64c) [ci: update pipeline icon to "cog"](https://github.com/flavorjones/loofah/commit/4e8a64ce9ff84054ad631a57082988ea151538a9)


</details>